StoreApp.BLL/Services/SaveXMLService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace StoreApp.BLL.Services
+{
+    public class SaveXMLService
+    {
+        public void SaveXML(HttpPostedFileBase file, string folder)
+        {
+            var pathStringContent = new DirectoryInfo(string.Format($"{AppDomain.CurrentDomain.BaseDirectory}Content\\"));
+            var pathStringContentXML = Path.Combine(pathStringContent.ToString(), "XMLUploads");
+            var pathStringXMLFolder = Path.Combine(pathStringContentXML.ToString(), folder);
```

Comment:
If you have already got a string you don't need to use **ToString()**.
In any case if you have an explanation why you did it I am ready to listen to you)

Reply:
It's old functional, I deleted this .cs